components/blitz/src/omero/client.java

```diff
@@ -350,11 +346,8 @@ private void init(Ice.InitializationData id) {
         optionallySetProperty(id, "omero.block_size", Integer
             .toString(omero.constants.DEFAULTBLOCKSIZE.value));
 
-        // Set the default encoding if this is Ice 3.5 or later
-        // and none is set.
-        if (Ice.Util.intVersion() >= 30500) {
-            optionallySetProperty(id, "Ice.Default.EncodingVersion", "1.0");
-        }
+        // Set the default encoding if none is set.
+        optionallySetProperty(id, "Ice.Default.EncodingVersion", "1.0");
```

Comment:
With Ice 3.6 and above being the only supported versions, we could also look at increasing the encoding version here as well.